heron/cli/src/python/main.py

```diff
@@ -111,30 +113,21 @@ def run(command, parser, command_args, unknown_args):
   :param unknown_args:
   :return:
   '''
-  status = 1
-  if command == 'activate':
-    status = activate.run(command, parser, command_args, unknown_args)
-
-  elif command == 'deactivate':
-    status = deactivate.run(command, parser, command_args, unknown_args)
-
-  elif command == 'kill':
-    status = kill.run(command, parser, command_args, unknown_args)
-
-  elif command == 'restart':
-    status = restart.run(command, parser, command_args, unknown_args)
-
-  elif command == 'submit':
-    status = submit.run(command, parser, command_args, unknown_args)
-
-  elif command == 'help':
-    status = cli_help.run(command, parser, command_args, unknown_args)
-
-  elif command == 'version':
-    status = version.run(command, parser, command_args, unknown_args)
-
-  return status
-
+  runners = {
+      'activate':activate,
+      'deactivate':deactivate,
+      'kill':kill,
+      'restart':restart,
+      'submit':submit,
+      'update':update,
+      'help':cli_help,
+      'version':version,
+  }
+
+  if command in runners.keys():
+    return runners[command].run(command, parser, command_args, unknown_args)
+  else:
```

Comment:
maybe passing `runners` to `create_parser`? For example, do `kill.create_parser(subparsers, self.runners)` and inside `kill` module associates string `kill` with a lambda function like the following:

````Python
runners['kill'] = lambda *args: run(*args)
````

Back in `main`, refactor `run` to:
````
def run(command, *rest_args):
    try:
        return self.runners[command](command, *rest_args)
    except:
        Log.error('Unknown subcommand: %s' % command)
        return 1
````